srcpkgs/omegat/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'omegat'
+pkgname=omegat
+version=4.3.3
+revision=1
+hostmakedepends="openjdk8"
+depends="virtual?java-runtime"
+short_desc="Multiplatform CAT tool application"
+maintainer="Jorge Santiago <jsantiago12tone@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://omegat.org/"
+distfiles="https://sourceforge.net/projects/omegat/files/OmegaT%20-%20Standard/OmegaT%20${version}/OmegaT_${version}_Source.zip"
+checksum=3cb39c9357aefe6efb274e1ee56f1ece57141b73a40a2416a7b405a6c8123184
+
+do_build() {
+	JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk ./gradlew -x test installDist
+}
+
+do_install() {
+	cd build/install/OmegaT
+	vmkdir usr/share/java/omegat
+	vcopy "*" usr/share/java/omegat
+	vinstall "${FILESDIR}/omegat.desktop" 644 usr/share/applications/
+	vinstall images/OmegaT.svg 644 usr/share/pixmaps/omegat.svg
+	vbin "${FILESDIR}/omegat.sh" omegat
+}
+
+post_install() {
+	vmkdir usr/share/doc/omegat
+	vcopy "build/install/OmegaT/docs/*" usr/share/doc/omegat
```

Comment:
```suggestion
	vcopy "build/install/OmegaT/docs/*" usr/share/doc/omegat
	chmod -R 644 ${DESTDIR}/usr/share/doc/omegat
```
this should fix the build error

Reply:
Instead of using vcopy to copy the entire directory, I'm now using vdoc on the actual documentation files that come specifically with the local build. Should fix it, indeed.

Reply:
Copying the directories with vcopy also result in the permission build error, and when trying to `chmod -R 644`, it gives me `chmod: cannot access '{path to file}': Permission denied`.